### ISSUE TITLE
Web support

### ISSUE THREAD

**Issue by mihaiav** [flipkart-incubator/madman-android#20]
we are looking for an IMA alternative. Our app is web based so I wonder, is there any plan for a web version ? Open source Media Ads Manager seem so scarce

**Comment by anirudhramanan** [comment]
Yes, there are no good alternatives available out there, that's the reason we built and open-sourced madman at the first place.

Coming to your question, we do not have any near term plan for a web based version. We did evaluate web-view vs native view rendering for ads, chose native because of the performance reasons. [Perf comparison](https://tech.flipkart.com/madman-an-open-source-ads-framework-for-android-99b5e57fdd30#5587)
